Dockerfile

```diff
@@ -9,9 +9,9 @@ COPY . /go/src/github.com/jacksontj/promxy
 RUN cd /go/src/github.com/jacksontj/promxy/cmd/promxy && CGO_ENABLED=0 go build -mod=vendor -tags netgo,builtinassets
 RUN cd /go/src/github.com/jacksontj/promxy/cmd/remote_write_exporter && CGO_ENABLED=0 go build -mod=vendor
 
-FROM alpine:3.18.11
-MAINTAINER Thomas Jackson <jacksontj.89@gmail.com>
-EXPOSE     8082
+FROM   alpine:3.21.3
+LABEL  org.opencontainers.image.authors="Thomas Jackson <jacksontj.89@gmail.com>"
```

Comment:
This change and the uppercase `AS` in line 1 fix warnings by Docker during the build. See https://docs.docker.com/reference/build-checks/maintainer-deprecated/ for reference.